crates/turbopack-node/src/source_map/mod.rs

```diff
@@ -311,6 +312,7 @@ impl StructuredError {
     }
 }
 
+#[instrument(skip_all)]
```

Comment:
Shouldn't the whole tracing impl be behind a feature?

Reply:
mostly concerned about performance, but according to the benchmarks it's fine, so all good